app/components/post_body_additional_content/post_body_additional_content.js

```diff
@@ -215,9 +212,9 @@ export default class PostBodyAdditionalContent extends PureComponent {
 
     generateToggleableEmbed = (isImage, isYouTube) => {
         let {link} = this.props;
-        const {shortenedLink} = this.state;
-        if (shortenedLink) {
-            link = shortenedLink;
+        const {expandedLink} = this.props;
+        if (expandedLink) {
+            link = expandedLink;
```

Comment:
In the `render` function you are already doing this.. if you pass the link as an argument you won't need to perform this check.